### DISCUSSION TITLE
Problems disabling -rpath when compiling curl7.66.0

### DISCUSSION THREAD

**Discussion by HanKunChai** [curl/curl#11281] [Q&A]
how to disable rpath in Makefile compile? I add --disable-rpath in config option but it didn't work.
In config.log file,  "--disable-rpath" is only shown in input param list. I cannot find any line that make this option valid.

I was told that setting" hardcode_into_libs=no" in libtools.m4 might help, but I set the option "hardcode_into_libs" to no under linux in Configure file, but it didn't work,

**Comment by bagder** [comment]
I'm curious how you get the rpath in there to begin with!
